### ISSUE TITLE
Z-summary interpreting

### ISSUE THREAD

**Issue by AKypR99** [smorabit/scWGCNA#88]
Hello,

Thanks for your amazing work.

I am currently using the hdWGCNA package on single cell data. So I get modules on a scRNAseq of cancer cells that I project on healthy cells in order to see what are the modules that are "absent" from the healthy cells and so "exclusive" to cancer cells.

To affirm this, I proceeded to the Z-statistics analysis which gave me two types of values which are the Z-summary.qual and Z.summary.pres. I thought I understood that, unlike Zsummary.pres which measures the preservation of the modules in the other dataset, Zsummary.qual is the quality of the individual modules themselves without taking into account the projection of the modules from the cancer cells onto the healthy cell dataset.

In a paper I read (https://onlinelibrary.wiley.com/doi/epdf/10.1002/jcb.28377), they describe on page 4 a segmentation of their modules according to different thresholds:

- Z-summary > 10 : Highly preserved modules
- 5 > Z-summary > 2 :
- Moderately preserved modules 2 > Z-summary : Very variable modules

In my case, since I want to see which modules are "exclusive" to cancer cells, I should prefer the modules being the most variable in the projection on normal cells and thus keep only the modules having a Z-summary.pres < 2 and exclude those having a Z-summary.pres > 10 since these last ones would be just as present in normal cells as in cancer cells, right ?

In addition to that, concerning the Z-summary.qual which is a quality criterion of the modules, do we agree that in all cases I should keep exclusively the modules with a Z-summary.qual > 10 and possibly those with a 5 > Z-summary.qual > 2?

**To summarize, the modules being "exclusive" to cancer cells would be those with a Z-summary.qual higher than 2 and a Z-summary.pres lower than 2, right?**

**Comment by smorabit** [comment]
Yes I think you have that right. I think this approach makes sense. However, there may also be a co-expression modules which are still present in the healthy cells, but exhibit some kind of differential expression pattern in the cancer cells, and these may also be of interest to you.

**Comment by AKypR99** [comment]
Hello Sam,

Thank you a lot for your answer.

By "_there may also be a co-expression modules which are still present in the healthy cells, but exhibit some kind of differential expression pattern in the cancer cells_", I am not sure I understood what you meant.

Indeed, what I want to remove is the modules that are as much "present" in the healthy cells as in the cancer cells (insisting on the fact that these modules should not be involved in the passage from cancer state 1 to cancer state 2 because finding them is the purpose of the study), in other words, the modules that are not exclusive to cancer cells.

In your answer, what you suggest is to do a DME (Differential module expression) analysis between cancer state 1 and 2 for the modules that were projected onto healthy cells and that turn out to be preserved in the latter? 

Or do a DME analysis between the healthy cells and the entire cancer cells (without distinguishing between the two cancer states). If it is the latter that you suggest, knowing that the healthy cell dataset is different from the one containing my cancer cells, I am not sure I can do a DME analysis on it with hdWGCNA, right?

Regards,
Anthony

**Comment by smorabit** [comment]
I think for your case your strategy with identifying the network in the cancer cells and performing the module preservation analysis in the healthy cells makes sense in terms of the conclusions that you are trying to make.
